src/shared.js

```diff
@@ -2072,33 +2072,21 @@ export const getParticipantSelection = async (filter) => {
      
 export const isDeviceMobile = /(tablet|ipad|playbook|silk)|(android(?!.*mobi))/i.test(window.navigator.userAgent) ||
     /Mobile|iP(hone|od)|Android|BlackBerry|IEMobile|Kindle|Silk-Accelerated|(hpw|web)OS|Opera M(obi|ini)/.test(window.navigator.userAgent) || window.innerWidth < 1300;
-export const replaceDateInputWithMaskedInput = (dateInput) => {
 
-  if (dateInput.type !== "date") throw new Error(`${dateInput} must be a input type="date"`);
+export const replaceDateInputWithMaskedInput = (dateInput) => {
   dateInput.type = "text";
   dateInput.placeholder = "mm/dd/yyyy";
   dateInput.maxLength = 10;
   dateInput.dataset.maskedInputFormat = "mm/dd/yyyy";
   dateInput.addEventListener("keypress", function (e) {
-    
-    if (e.keyCode < 47 || e.keyCode > 57) {
+    // Only allows number inputs and deletes
+    if (e.keyCode < 48 || e.keyCode > 57) {
```

Comment:
Did you change 47 to 48 because the 48 keycode is where the numbers for keycode starts? I noticed from the keycode list that 48 is "0" and 57 is "9". 

Reference: [Keycode List](https://blogs.longwin.com.tw/lifetype/key_codes.html)

Reply:
Key 47 is "/". I removed it so that there's no need to type it manually. In the changed code, when you type for example "01311970", you get "01/31/1970". Plus, during typing, all other characters (besides numbers 0-9) at position 3 and 6  will be changed to "/".  For example, if you type "02@", you get "02/" because "@" is ignored and "/" is added, instead.